packages/db/tests/search/test-queries.js

```diff
@@ -104,6 +126,10 @@ test('convertQueryStringToDbQueryParameters correctly converts sortby error para
         column: 'error.Error',
         order: 'asc',
       },
+      {
```

Comment:
Nit: we should probably either modify this test name *or* explicitly test that this parameter is *last*, as that's a required behavior to ensure that cumulus_id sort acts only as a default *or* tiebreaker in cases where there's a user  sort key specified.